Libplanet/Blocks/MultipleTxsOfSameSignerException.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+
+namespace Libplanet.Blocks
+{
+    /// <summary>
+    /// The exception that is thrown when the namespace of
+    /// <see cref="Block{T}"/> has multiple transactions of the same signer.
+    /// </summary>
+    [Serializable]
+    public sealed class MultipleTxsOfSameSignerException : Exception
```

Comment:
Would there be a better name for this? A type name having a preposition like “of” seems quite awkward to me.

Reply:
I renamed it to `SingersHaveMoreThanOneTxsException`.